src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -144,99 +144,7 @@
-          </v-col>
-        </v-row>
-      </section>
-
-      <section v-if="allResolutionValidationItemsInvalid"
-        class="section-container invalid-section rounded-bl-0"
-      >
-        <v-icon color="error">mdi-information-outline</v-icon>
-        <span class="error-text ml-1 mr-2">This step is unfinished.</span>
-        <router-link
-          :to="{ path: `/${RouteNames.CREATE_RESOLUTION}` }"
-        >Return to this step to finish it</router-link>
-      </section>
-
-      <section id="resolution-summary-section-3"
-               v-if="isTypeCoop && !allResolutionValidationItemsInvalid"
-               class="section-container rounded-bl-0"
-               :class="{ 'invalid-section': !getCreateResolutionStep.validationDetail.valid }"
-      >
-        <span v-if="!getCreateResolutionStep.validationDetail.valid">
-          <v-icon color="error">mdi-information-outline</v-icon>
-          <span class="error-text ml-1 mr-2">This step is unfinished.</span>
-          <router-link
-            :to="{ path: `/${RouteNames.CREATE_RESOLUTION}` }"
-          >Return to this step to finish it</router-link>
-        </span>
-        <v-row class="ml-2 mt-6" no-gutters>
-          <v-col md="3">
-            <strong>Resolution Date</strong>
-          </v-col>
-          <v-col md="9" class="ml-n1">
-            {{resolutionDate}}
-          </v-col>
-        </v-row>
-        <v-row class="ml-2 mt-5" no-gutters>
-          <v-col md="3">
-            <strong>Resolution Text</strong>
-          </v-col>
-          <v-col md="9" class="ml-n1">
-            <v-lazy>
-              <v-textarea
-                id="resolution-text"
-                rows="1"
-                auto-grow
-                readonly
-                filled
-                background-color="white"
-                v-model="resolutionText">
-              </v-textarea>
-            </v-lazy>
-          </v-col>
-        </v-row>
-        <v-row class="ml-2 mt-3" no-gutters>
-          <v-col md="3">
-            <strong>Signing Party</strong>
-          </v-col>
-          <v-col md="9" class="ml-n1">
-            {{signingParty}}
-          </v-col>
-        </v-row>
-        <v-row class="ml-2 mt-5" no-gutters>
-          <v-col md="3">
-            <strong>Date Signed</strong>
-          </v-col>
-          <v-col md="9" class="ml-n1">
-            {{signingDate}}
-          </v-col>
-        </v-row>
-        <v-row v-if="getCreateResolutionStep.resolutionConfirmed" class="ml-1 mt-5" no-gutters>
-          <v-col md="auto" class="mr-2">
-            <v-icon class="upload-success-chk" color="successCheckmark">mdi-check</v-icon>
-          </v-col>
-          <v-col md="11" id="file-name-col">
-            <span>{{ getCreateResolutionResource.confirmSection.reviewSummaryText }}</span>
-          </v-col>
-        </v-row>
-      </section>
-    </v-card>
+    <CompleteResolutionSummary></CompleteResolutionSummary>
```

Comment:
Much cleaner! 

We typically self close components, unless they're being extended using slots or something. Not sure if thats just our approach or whats widely preferred haha. 

ie `<CompleteResolutionSummary />`

Reply:
yeah i like that too...looks cleaner.  will get that updated

Reply:
updated to be self closing